examples/pages/components/TimePicker.tsx

```diff
@@ -19,13 +21,28 @@ class TimePickerExample extends React.PureComponent<{}, {time: string | null}> {
     render() {
         return (
             <TimePicker
+                headerTemplate={
+                    <ButtonGroup spaces="compact" align="center">
+                        <Button size="small" text="In 30 min" style="hollow" onClick={() => false} />
+                        <Button size="small" text="In 1 hr" style="hollow" onClick={() => false} />
+                        <Button size="small" text="In 2 hr" style="hollow" onClick={() => false} />
+                        <Button size="small" text="In 5 hr" style="hollow" onClick={() => false} />
+                    </ButtonGroup>
+                }
+                footerTemplate={<div>Footer</div>}
                 value={this.state.time}
                 onChange={(time) => {
                     this.setState({time});
                 }}
                 allowSeconds
                 label="This is Label"
                 info="This is info"
+                inputWrapper={{kind: 'custom', component: ({input}) => (
```

Comment:
adjust the styles so it looks well when a custom wrapper is used

![image](https://github.com/user-attachments/assets/971cb00a-40a7-4634-8983-8b4cd5475f06)

Reply:
feel free to undo this `inputWrapper` demo, I just added it so you can reproduce it breaking